example/index.html

```diff
@@ -1,110 +1,192 @@
 <!DOCTYPE html>
 <html class="no-js">
+
 <head>
     <title>Quicktube</title>
-    <link rel="stylesheet" href="../dist/quicktube.css"/>
+    <link rel="stylesheet" href="../dist/quicktube.css" />
```

Comment:
Yes please, then in the `build` script (https://github.com/springload/Quicktube.js/blob/master/package.json#L12), add a step that copies the file from `src` to `dist`.